### ISSUE TITLE
Add native support for stacked charts

### ISSUE THREAD

**Issue by stsdc** [lcallarec/live-chart#33]
I'm using stacked chart to display CPU usage and I was wondering if would it be possible to add this as a native feature?

https://github.com/stsdc/monitor/blob/b69fdad44329b66ce8ee479af9f64f6c98f183de/src/Views/SystemView/SystemCPUView.vala#L94-L107

![image](https://user-images.githubusercontent.com/6031763/157544136-3a3517d0-5f87-4201-a2d1-fe28ecbe1f85.png)

**Comment by lcallarec** [comment]
Yes, I think I can do that, it's pretty close to stacked bar feature.
